src/UndoRedo.spec.ts

```diff
@@ -2,4 +2,130 @@
 declare var require: NodeRequire;
 
 import test = require('tape');
-import undoRedo from './';
+import undoRedoStack from './';
```

Comment:
along with this, can you also add:
`import { UndoRedo } from './'`
because it turns you actually do use UndoRedo class in your function parameters.